### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by timea-techgirl**
English translation update and grammatical corrections.

### CODE REVIEW

**Comment by liudongmiao** [comment]
@liyufan @aviraxp @TacoTheDank @vvb2060 please help review.

@timeakesmarky please add your name to field of `brevent_about_translator`.

File: brevent/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -180,18 +180,18 @@
...
+    <string name="brevent_optimize_wechat_label">Add to Battery Optimization whitelist, make it receive messages as far as possible</string>
     <string name="brevent_optimize_audio">Optimize music player</string>
-    <string name="brevent_optimize_audio_label">Won\'t be brevented while playing with notification, won\'t be force-stopped while pausing with notification\n\nNot works on some ROMs. May cause some samsung devices reboot</string>
+    <string name="brevent_optimize_audio_label">Won\'t be brevented while playing with notification, won\'t be force-stopped if paused with notification\n\nDoes not work on some ROMs. May cause some samsung devices to reboot</string>
     <string name="brevent_auto_aggressive">Auto aggressive</string>
-    <string name="brevent_auto_aggressive_label">If app is ran with notifications as service, set it to aggressive.</string>
+    <string name="brevent_auto_aggressive_label">If app is run with notifications as a service, set it to aggressive.</string>
     <string name="show_os_apps">Show OS apps</string>
```
Comment (aviraxp) [inline brevent/src/main/res/values/strings.xml:195]: runs as a service with notifications

**Review by aviraxp — CHANGES_REQUESTED** [review]

File: brevent/src/main/res/values/strings.xml (hunk 1 above)
Comment (timea-techgirl) [reply brevent/src/main/res/values/strings.xml:195]: Changed, thanks.

**Review by aviraxp — APPROVED** [review]

File: brevent/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -358,13 +358,13 @@
...
     <bool name="support_wechat">false</bool>
 
-    <string name="new_version_available">There is a updated version \"%s\".</string>
+    <string name="new_version_available">There is an updated version \"%s\".</string>
     <string name="latest_version">You have installed the latest version.</string>
     <string name="beta_version">You have installed a beta version.</string>
     <string name="update_version">Update</string>
 
     <string name="legal_drawer">Terms &amp; Privacy</string>
-    <string name="legal_short">You must accept Terms &amp; Privacy to use Brevent. In short, Brevent will collect Anonymous Log Data via Google Analytics for Firebase.</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values/strings.xml:367]: `Google Analytics for Firebase` is the product name for Firebase.

File: brevent/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -180,18 +180,18 @@
     <string name="brevent_language_auto">Auto</string>
     <string name="brevent_theme">Theme</string>
     <string name="show_framework_apps">Show core apps</string>
-    <string name="show_framework_apps_label">All apps have the same signature with framework</string>
+    <string name="show_framework_apps_label">All framework apps have the same signature</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values/strings.xml]: This doesn't explain framework apps.

Framework apps are a category in Brevent, it means apps have the same signature with Android Framework (with the package name `android`).

Comment (timea-techgirl) [reply brevent/src/main/res/values/strings.xml]: Oh ok, I got it. I have changed it to an understandable way now.

**Comment by liudongmiao** [comment]
https://jianyu.io/br/alpha/ please download and install to verify.

**Comment by timea-techgirl** [comment]
> https://jianyu.io/br/alpha/ please download and install to verify.

Oh, thanks. I was waiting for this. I got a better context to the strings and made final changes. :)

**Comment by timea-techgirl** [comment]
The string ""Be careful before breventing system apps or pay %1$s brefoil to dismiss this safe warning" hasn't been changed in the test version.

**Comment by liudongmiao** [comment]
update alpha to 3.7.2+3: https://jianyu.io/br/alpha/

And, please add your name to https://github.com/brevent/Brevent/blob/translation/CONTRIBUTORS.md.

**Comment by timea-techgirl** [comment]
Done, looks fine.

**Comment by liudongmiao** [comment]
Please contact via email for contributor promo code.

**Pull request merged**